CRM/Contact/Form/CustomData.php

```diff
@@ -224,7 +224,7 @@ public function setDefaultValues() {
           TRUE,
           NULL,
           FALSE,
-          TRUE,
+          CRM_Core_Permission::EDIT,
```

Comment:
Is this right?

Reply:
I'm presuming that the user needs `EDIT` permission in order to use a form, yes?

Reply:
Oh I didn't notice yet the called function itself has changed. I see.

Reply:
There's a couple calls in extensions but the only one that I see that uses this parameter is drupal 7 views, but it passes FALSE.